modules/verifying-cluster--install-oci.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_oci/installing-oci-assisted-installer.adoc [Using the Assisted Installer to install a cluster on OCI]
+
+:_mod-docs-content-type: PROCEDURE
+[id="verifying-cluster--install-oci_{context}"]
+= Verifying a succesful cluster installation on OCI
+
+Verify that your cluster was installed and is running effectively on {oci-first}. To succesfully install a cluster and deploy it on {oci-first}, you must create specific {oci} infrastucture resources. Additionally, you must ensure that the Assisted Installer can access these {oci} infrastucture resources.
+
+.Prerequisites
+
+* You completed all the Assisted Installer steps for installing a cluster on {oci}.
+
+.Procedure
+
+. From the Hybrid Cloud Console, go to **Clusters > Assisted Clusters** and select your cluster’s name. 
+
+. Check that the **Installation progress** bar is at **100%** and a message opens that indicates `Installation completed successfully`.
+
+. To access the {product-title} web console, click on the provided Web Console URL.
+
+. Go to the **Nodes** menu page. 
```

Comment:
Should be "Go to the Compute -> Nodes" in Openshift Dashboard UI

Reply:
Updated. TY!